bin/hdfcoinc/pycbc_fit_sngls_by_template

```diff
@@ -311,8 +318,12 @@ for tnum in trange:
                                                     (tnum - tmin, tmax - tmin))
 
 logging.info("Calculating median sigma for each template")
-median_sigma = [np.sqrt(np.median(trigf[args.ifo + '/sigmasq'][reg])) for reg in
-                    trigf[args.ifo + '/sigmasq_template'][:]]
+
+sigma_regions = trigf[args.ifo + '/sigmasq_template'][:]
+median_sigma = []
+for i, reg in enumerate(sigma_regions):
+    strigs = trigf[args.ifo + '/sigmasq'][reg]
+    median_sigma.append(np.median(strigs)**0.5)
```

Comment:
what is the `i` variable used for in this case? I'm not sure how this is faster than the list comprehension

Reply:
Ok, I think I might have used it for vebose timing at one part. Probably not needed anymore.

Reply:
I've now removed this